File: app/src/main/java/com/rakuishi/ok/model/Repo.java
package com.rakuishi.ok.model;

import com.google.gson.annotations.SerializedName;

public class Repo {

    @SerializedName("id")
    public int id;

    @SerializedName("name")
    public String name;

    @SerializedName("description")
    public String description;

    @SerializedName("updated_at")
    public String updatedAt;

    @SerializedName("html_url")
    public String url;
}

File: app/src/main/java/com/rakuishi/ok/data/Repo.kt
package com.rakuishi.ok.data

import com.google.gson.annotations.SerializedName

data class Repo(
    @SerializedName("id")
    val id: Int,
    @SerializedName("name")
    val name: String,
    @SerializedName("description")
    val description: String,
    @SerializedName("updated_at")
    val updatedAt: String,
    @SerializedName("html_url")
    val url: String
)